core/src/main/java/com/novoda/noplayer/internal/exoplayer/forwarder/EventInfoForwarder.java

```diff
@@ -58,6 +57,15 @@ public void onPlayerStateChanged(boolean playWhenReady, int playbackState) {
     }
 
     @Override
+    public void onRepeatModeChanged(@com.google.android.exoplayer2.Player.RepeatMode int repeatMode) {
+        HashMap<String, String> callingMethodParameters = new HashMap<>();
```

Comment:
Can we do this?

`Map<String, String> callingMethodParameters = new HashMap<>(1);`